src/geff/metadata_schema.py

```diff
@@ -135,6 +135,53 @@ class DisplayHint(BaseModel):
     )
 
 
+class PropMetadata(BaseModel):
+    """Metadata describing a property in the geff graph."""
+
+    identifier: str
+    dtype: str  # TODO: investigate how other packages deal with data types
+    unit: str
```

Comment:
```suggestion
    unit: str | None = None
```
I think we need to leave this as optional